src/pages/publication/contributors_tab/components/AddContributorModal.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC, useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+import { Button, Typography } from '@material-ui/core';
+
+import { getAuthorities } from '../../../../api/authorityApi';
+import SearchBar from '../../../../components/SearchBar';
+import { searchFailure } from '../../../../redux/actions/searchActions';
+import { Authority, emptyAuthority } from '../../../../types/authority.types';
+import { debounce } from '../../../../utils/debounce';
+import AuthorityCard from '../../../user/authority/AuthorityCard';
+
+const StyledClickableDiv = styled.div`
+  cursor: pointer;
+  align-items: center;
+  background-color: ${({ theme }) => theme.palette.box.main};
+  padding-right: 0.5rem;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 1rem;
+`;
+
+interface DisplayValue {
+  searchTerm: string;
+  results: number;
+}
+
+interface AddContributorModalProps {
+  addAuthor: (selectedAuthor: Authority) => void;
+}
+
+const AddContributorModal: FC<AddContributorModalProps> = ({ addAuthor }) => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation('publication');
+
+  const [matchingAuthorities, setMatchingAuthorities] = useState<Authority[]>();
+  const [selectedAuthor, setSelectedAuthor] = useState<Authority>(emptyAuthority);
```

Comment:
Burde ikke verdien være `null` før man har valgt noen? `emptyAuthority` kan kanskje være ett tilsynelatende gyldig valg også?

Reply:
vi sjekker på scn for selectedAuthor da, så det vil jo ikke være gyldig. i og med at vi har brukt emptyBlabla i resten av appen i stedet for å gjøre alle typer nullable, så tenker jeg det er mest logisk her også